Readme

annehong opened the pull request.

antelopeb commented:
@annehong -- I think this sounds pretty good. Steps 9 and 12 could probably be condensed into one (not 9 through 12, just 9 AND 12). That is, you should create that directory `working-dir/component/markup-kit` first, then unzip the files inside it.

Also, we might consider putting this into a separate README file. Maybe README-contribute.md?

@davodey, @dengeist -- thoughts?

annehong commented:
Hi @antelopeb,

I made the changes as recommended. I agree that it should go into a separate readme file because it was slightly confusing that some of the steps overlapped with the current steps in README file. I removed them and pasted it into README-contribute.md file and consolidated steps 9-12 into one step, and removed former steps 10 and 11.

Please let me know if this is okay?

Thanks,
Anne
